test/test_api_query.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+from fastapi.testclient import TestClient
+
+
+def test_example_model(monkeypatch):
```

Comment:
Hi @skirdey, are you still working on it? It seems that the tests are failing. Could it be possible due to unintended side-effects of applying `monkeypath` here?